pkg/webhook/testdata/app/installations/0/access_tokens

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
did not find where this is used

Reply:
It's used in the token exchange transport. Changed this from 0 -> 1111 to make it more obvious how this is plumbed through.